pennylane/qchem/factorization.py

```diff
@@ -124,43 +126,39 @@ def factorize(two_electron, tol_factor=1.0e-5, tol_eigval=1.0e-5):
         - Reshape the :math:`n \times n \times n \times n` two-electron tensor to a
           :math:`n^2 \times n^2` matrix where :math:`n` is the number of orbitals.
 
-        - Diagonalize the resulting matrix and keep the :math:`r` eigenvectors that have
-          corresponding eigenvalues larger than a threshold.
+        - Decompose the resulting matrix either via Cholesky decomposition or
+          via eigenvalue decomposition.
 
-        - Multiply the eigenvectors by the square root of the eigenvalues to obtain
-          matrices :math:`L^{(r)}`.
+        - In the Cholesky decomposition, we build matrices :math:`L^{(r)}` as
```

Comment:
Cholesky is not the default, so better to explain eigenvalue decomposition first.